engine/src/main/java/org/terasology/rendering/opengl/AbstractFBOsManager.java

```diff
@@ -44,11 +44,11 @@
  * rendering engine.
  * <p>
  * Default FBOs:
- * sceneOpaque:  Primary FBO: most visual information eventually ends up here
- * sceneOpaquePingPong:  The sceneOpaque FBOs are swapped every frame, to use one for reading and the other for writing
+ * writeOnlyGBuffer: this is the primary FBO and its attachments are the target of a good number of rendering operations, eventually storing most information needed for the deferred rendering.
+ * readOnlyGBuffer: this FBO holds attachments that are used as input textures by a number of rendering stages.
+ * Note that these two FBOs might get swapped during the DAG creation stage, on a node's request. However, this is handled transparently and getWriteOnlyFbo() will always return the FBO appropriate for writing.
```

Comment:
Note that these two FBOs **may be swapped** on a node's request **(ping-pong technique). See the DisplayResolutionDependentFboManager class for more details.**